src/main.rs

```diff
@@ -1,3 +1,5 @@
+use iroh_rpc;
+
 fn main() {
```

Comment:
this should move to `iroh/src/main.rs`

Reply:
🤦‍♀️ did a bad copy paste job

thanks!